src/BusinessLogic/Resources/scss/icons.scss

```diff
@@ -0,0 +1,18 @@
+/* Material icons */
+
+//noinspection CssNoGenericFontName
+#pl-page .material-icons {
+  font-family: 'Material Icons';
```

Comment:
Codacy found an issue: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
This is an icon font. It does not have a fallback.